src/main/java/slimeknights/tconstruct/library/modifiers/data/ModifierMaxLevel.java

```diff
@@ -40,4 +42,8 @@ public void set(EquipmentSlot slot, float level) {
       }
     }
   }
+
+  public static float getStat(LivingEntity living, TinkerDataCapability.ComputableDataKey<ModifierMaxLevel> key) {
+    return living.getCapability(TinkerDataCapability.CAPABILITY).resolve().map(data -> data.get(key)).map(ModifierMaxLevel::getMax).orElse(0f);
```

Comment:
Don't forget the function docs here. Does not need to be long, just briefly explain when this is useful.

Up to you whether you want to include param in the doc string.